### PR TITLE
Implement TagSort command

### LINKED ISSUE #115: As a user, I want to sort the tags by its tag name
so that I can view the tags in alphabetical order

### PR DESCRIPTION
**Pull request by siqiang-ng**

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -192,6 +196,24 @@ public void updateFilteredTagList(Predicate<Tag> predicate) {
         filteredTags.setPredicate(predicate);
     }
 
+    //=========== Filtered Person List Accessors =============================================================
```
Comment (khinkhinn) [inline src/main/java/seedu/address/model/ModelManager.java]: should this be Sorted Tag List Accessors?

**Review by akgrenSoar — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (siqiang-ng) [reply src/main/java/seedu/address/model/ModelManager.java]: ohh yep thanks

**Review by JoeyToh — COMMENTED** [review]
lgtm

**Review by khinkhinn — APPROVED** [review]

**Pull request merged**